core/src/commonMain/kotlin/org/mobilenativefoundation/store/core5/StoreData.kt

```diff
@@ -0,0 +1,35 @@
+package org.mobilenativefoundation.store.core5
+
+
+/**
+ * An interface that defines items that can be uniquely identified.
+ * Every item that implements the [StoreData] interface must have a means of identification.
+ * This is useful in scenarios when data can be represented as singles or collections.
+ */
+
+interface StoreData<out Id : Any> {
```

Comment:
does every store response need to be wrapped in a store data?

Reply:
No, only required for the paging mechanism